### PR TITLE
adding build command

### PR DESCRIPTION
**Pull request by deanshub**
### 🔦 Summary
adding build command to yoshi-flow-editor

### CODE REVIEW

File: packages/create-yoshi-app/templates/out-of-iframe/javascript/dev/wix-private.mock.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ if (window.Wix && window.Wix.Utils.getViewMode() === 'standalone') {
       });
     }
 
+    addEventListener(eventName, cb) {
```
[the remark this answers is not in the document]
Comment (deanshub) [reply packages/create-yoshi-app/templates/out-of-iframe/javascript/dev/wix-private.mock.js]: resolved in flow-editor-wix-mock branch

File: packages/yoshi-flow-editor-runtime/src/react/PublicDataRenderProp.tsx [hunk 2]
```diff
@@ -13,15 +13,6 @@ export class PublicData extends React.Component<
     return (
       <PublicDataContext.Consumer>
         {publicData => {
-          // If we are in editor mode (readyPromise)
```
[the remark this answers is not in the document]
Comment (deanshub) [reply packages/yoshi-flow-editor-runtime/src/react/PublicDataRenderProp.tsx]: suspense takes care of it

File: packages/create-yoshi-app/templates/out-of-iframe/javascript/src/platform/viewerScript.js [hunk 3]
```diff
@@ -12,6 +12,8 @@ export const createControllers = controllerConfigs => {
   const [controllerConfig] = controllerConfigs;
   const { appParams, platformAPIs, wixCodeApi, csrfToken } = controllerConfig;
 
+  initializeExperiments();
```
[the remark this answers is not in the document]
Comment (deanshub) [reply packages/create-yoshi-app/templates/out-of-iframe/javascript/src/platform/viewerScript.js]: moved to create controllers only for now

File: packages/create-yoshi-app/templates/out-of-iframe/typescript/dev/server.js [hunk 4]
```diff
@@ -34,6 +34,17 @@ app.use('/settingsPanel', (req, res) => {
   res.send(renderVM('./src/templates/settingsPanel.vm'));
 });
 
+const state = {};
+
```
[the remark this answers is not in the document]
Comment (deanshub) [reply packages/create-yoshi-app/templates/out-of-iframe/typescript/dev/server.js]: removed

File: packages/yoshi-flow-editor-runtime/src/types.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+// Names of properties in T with types that include undefined
+type OptionalPropertyNames<T> = {
```
Comment (ranyitz) [inline packages/yoshi-flow-editor-runtime/src/types.ts]: I think that we can remove this file whatsoever as we don't use these types in the codebase

Comment (ranyitz) [reply packages/yoshi-flow-editor-runtime/src/types.ts]: Fixed on - https://github.com/wix/yoshi/pull/1612/commits/bc21916df0e21481534504740276cac5e4abf772

**Pull request merged**